decode.go

```diff
@@ -1266,6 +1266,22 @@ func unquoteBytes(s []byte) (t []byte, ok bool) {
 				b[w] = '\t'
 				r++
 				w++
+			case 'x':
+				if (r+2) < len(s) && s[r+1] == '1' && (s[r+2] == 'b' || s[r+2] == 'B') {
+					// Interpret as raw escape sequence
+					b[w] = '\x1b'
+					r += 3
+					w++
+					// Parse/allow a raw sequence size of 14 characters + '"'
+					for r < len(s) && r < 16 {
```

Comment:
- Do we need to parse the entire escape sequence? I think we only need to parse the escape character `\x1b`.
- So this inner loop seems not needed at all. The same is done in the outer loop anyway.
-  Furthermore, why not add generic `\x` support for escaping arbitrary hex bytes, not just the escape byte 0x1b? After all, this is a generic JSON parsing library, it shouldn't need to know about terminal escape sequences or about micro.

Reply:
> So this inner loop seems not needed at all. The same is done in the outer loop anyway.

You're right. I focused too much on the `\u` part within the loop, that I didn't realize that the rest of the ASCII characters including the string end will be properly handled already.

> Furthermore, why not add generic \x support for escaping arbitrary hex bytes, not just the escape byte 0x1b?

Fully ACK, it's more generic and allows even more scenarios. I will drop the inner loop and adapt the handling to support the escaped hex value.